webcompat/views.py

```diff
@@ -265,14 +265,8 @@ def create_issue():
     # Form Prefill section
     if request_type == 'prefill':
         form_data = prepare_form(request)
-
-        if ab_active('exp') == 'form-v2':
-            bug_form = get_form(form_data, form=FormWizard)
-            pagetitle = "New Issue |"
-        else:
-            bug_form = get_form(form_data)
-            pagetitle = "New Issue"
-
+        bug_form = get_form(form_data, form=FormWizard)
+        pagetitle = "New Issue"
```

Comment:
Should the title still remain as `pagetitle = "New Issue |"` here? I think this was added for ga tracking to differentiate the reports from old and new form (not sure if this being tracked still though)

Reply:
I asked Christos, and he said it's fine to remove it.